ingestion/src/metadata/ingestion/source/pipeline/fivetran/metadata.py

```diff
@@ -83,12 +72,26 @@ def create(
             )
         return cls(config, metadata)
 
-    def get_connections_jobs(self, pipeline_details: FivetranPipelineDetails):
+    def get_connections_jobs(
+        self,
+        pipeline_details: FivetranPipelineDetails,
+        source_url: Optional[SourceUrl] = None,
+    ) -> list[Task]:
         """Returns the list of tasks linked to connection"""
         return [
             Task(
                 name=pipeline_details.pipeline_name,
                 displayName=pipeline_details.pipeline_display_name,
+                sourceUrl=source_url,
+                fullyQualifiedName=None,
```

Comment:
Do we need to pass those values as `None` ?
They are by default `None` only.

Reply:
We can have either, mypy was crying, that's why i added those, it can either be all the args, or I can add ignore as well.